.github/workflows/main.yml

```diff
@@ -945,123 +953,136 @@ jobs:
+        run: ant $OPTS -f platform/editor.mimelookup test
 
       - name: platform/editor.mimelookup.impl
-        run: ant $OPTS -Dvanilla.javac.exists=true -f platform/editor.mimelookup.impl test
+        run: ant $OPTS -f platform/editor.mimelookup.impl test
 
       - name: platform/favorites
-        run: .github/retry.sh ant $OPTS -Dvanilla.javac.exists=true -f platform/favorites test
+        run: .github/retry.sh ant $OPTS -f platform/favorites test
 
       - name: platform/javahelp
-        run: ant $OPTS -Dvanilla.javac.exists=true -f platform/javahelp test-unit
+        run: ant $OPTS -f platform/javahelp test-unit
 
       - name: platform/keyring.fallback
-        run: ant $OPTS -Dvanilla.javac.exists=true -f platform/keyring.fallback test
+        run: ant $OPTS -f platform/keyring.fallback test
 
       - name: platform/keyring.impl
-        run: ant $OPTS -Dvanilla.javac.exists=true -f platform/keyring.impl test
-
-      - name: platform/lib.uihandler
-        run: ant $OPTS -Dvanilla.javac.exists=true -f platform/lib.uihandler test
+        run: ant $OPTS -f platform/keyring.impl test
 
       - name: platform/libs.javafx
-        run: ant $OPTS -Dvanilla.javac.exists=true -f platform/libs.javafx test
+        run: ant $OPTS -f platform/libs.javafx test
 
       - name: platform/libs.junit4
-        run: ant $OPTS -Dvanilla.javac.exists=true -f platform/libs.junit4 test
+        run: ant $OPTS -f platform/libs.junit4 test
 
       - name: platform/masterfs
-        run: .github/retry.sh ant $OPTS -Dvanilla.javac.exists=true -f platform/masterfs test
+        run: .github/retry.sh ant $OPTS -f platform/masterfs test
 
       - name: platform/masterfs.linux
-        run: ant $OPTS -Dvanilla.javac.exists=true -f platform/masterfs.linux test
+        run: ant $OPTS -f platform/masterfs.linux test
 
       - name: platform/o.n.core
-        run: ant $OPTS -Dvanilla.javac.exists=true -f platform/o.n.core test-unit
+        run: ant $OPTS -f platform/o.n.core test-unit
 
       - name: platform/o.n.swing.outline
-        run: ant $OPTS -Dvanilla.javac.exists=true -f platform/o.n.swing.outline test
+        run: ant $OPTS -f platform/o.n.swing.outline test
 
       - name: platform/o.n.swing.tabcontrol
-        run: ant $OPTS -Dvanilla.javac.exists=true -f platform/o.n.swing.tabcontrol test
+        run: ant $OPTS -f platform/o.n.swing.tabcontrol test
+
+      - name: Set up JDK 8 for incompatibe tests
+        uses: actions/setup-java@v3
+        with:
+          java-version: 8
+          distribution: ${{ env.default_java_distribution }}
 
-      # required by netbinox tests
+      # TODO fix JDK 11 incompatibilities
+      - name: platform/lib.uihandler
+        run: ant $OPTS -f platform/lib.uihandler test
+
+      - name: platform/openide.text
+        run: .github/retry.sh ant $OPTS -f platform/openide.text test
+
+      - name: platform/openide.util.ui
+        run: ant $OPTS -f platform/openide.util.ui test
```

Comment:
`openide.util.ui` has some failures due to JDK xml lib updates. One is a whitespace problem since whitespace parsing changed and is parsed into nodes, the other is a formatting problem. The formatting test is probably not easily fixable and might not be worth it to fix. I have a patch for the other failure locally.

Forgot specifics about the other two modules.